- en: scipy.odr.OdrStop
  id: totrans-0
  prefs:
  - PREF_H1
  type: TYPE_NORMAL
  zh: scipy.odr.OdrStop
- en: Original text：[https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.odr.OdrStop.html#scipy.odr.OdrStop](https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.odr.OdrStop.html#scipy.odr.OdrStop)
  id: totrans-1
  prefs:
  - PREF_BQ
  type: TYPE_NORMAL
  zh: 原文：[https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.odr.OdrStop.html#scipy.odr.OdrStop](https://docs.scipy.org/doc/scipy-1.12.0/reference/generated/scipy.odr.OdrStop.html#scipy.odr.OdrStop)
- en: '[PRE0]'
  id: totrans-2
  prefs: []
  type: TYPE_PRE
  zh: '[PRE0]'
- en: Exception stopping fitting.
  id: totrans-3
  prefs: []
  type: TYPE_NORMAL
  zh: 异常停止拟合。
- en: You can raise this exception in your objective function to tell [`odr`](odr-function.html#scipy.odr.odr
    "scipy.odr.odr") to stop fitting.
  id: totrans-4
  prefs: []
  type: TYPE_NORMAL
  zh: 你可以在你的目标函数中引发这个异常，告诉[`odr`](odr-function.html#scipy.odr.odr "scipy.odr.odr")停止拟合。
- en: '[PRE1]'
  id: totrans-5
  prefs: []
  type: TYPE_PRE
  zh: '[PRE1]'
- en: Exception.with_traceback(tb) – set self.__traceback__ to tb and return self.
  id: totrans-6
  prefs: []
  type: TYPE_NORMAL
  zh: '`Exception.with_traceback(tb)` – 设置`self.__traceback__`为`tb`并返回`self`。'
